components/dropdown/src/auro-dropdown.js

```diff
@@ -236,6 +277,12 @@ export class AuroDropdown extends LitElement {
     if (changedProperties.has('mobileFullscreenBreakpoint')) {
       this.bibContent.mobileFullscreenBreakpoint = this.mobileFullscreenBreakpoint;
     }
+
+    if (this.triggerContentSlot) {
```

Comment:
Do we need to do this check EVERY time the component gets updated?

Reply:
I changed it to limit it to run that block only when there is no changed property. (called by `reuqestUpdate` function